CHANGELOG.md

```diff
@@ -19,13 +19,16 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * Fixed accessing josepy contents through acme.jose when the full acme.jose
   path is used.
 * Clarify behavior for deleting certs as part of revocation.
+* Rename old,default.conf to old-and-default.conf to address breakages
+  from venv downloading the latest pip
```

Comment:
nit: Can we generalize this and say something like:

> Rename old,default.conf to old-and-default.conf to address commas in filenames breaking recent versions of pip.

Reply:
Yep, great idea